src/Sql/Sql.Test/ScenarioTests/ManagedDatabaseLogReplayScenarioTest.ps1

```diff
@@ -380,8 +334,8 @@ function Test-PipingCompleteCancelManagedDatabaseLogReplay
 		
 		try {
 			$dbRemoved = Get-AzSqlInstanceDatabase `
-				-ResourceGroupName $rgName `
-				-InstanceName $managedInstance `
+				-ResourceGroupName $resourceGroupName `
+				-InstanceName $managedInstanceName `
```

Comment:
According to the error message, the test case fails in the line of "Assert-Null $dbRemoved“，where $dbRemoved is not null.

Reply:
Currently I can see from the error log is that the db that should be removed still can be retrieved by Get-AzSqlInstanceDatabaseLogReplay.
There are something can be done for the issue check.
- please make sure the test case does run with no error in windows. Sometime the exception is swallowed by catch.
- please also check whether the path join, if applicable, in the implementation of the cmdlets used in the test case also works well in Linux.